packages/synapse-react-client/src/testutils/vitest.setup.ts

```diff
@@ -0,0 +1,2 @@
+// import jest-dom for Testing Library matchers (compatible with Vitest)
+import '@testing-library/jest-dom/vitest'
```

Comment:
Provides useful matchers, [e.g. `.toBeInTheDocument()`](https://github.com/testing-library/jest-dom). They provide a Vitest implementation, so use it here.